maintenance/aws-janitor/BUILD

```diff
@@ -37,3 +37,4 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
```

Comment:
you should make your editor run buildifier on write. :)

Reply:
nobody has a good extension for bazel yet, i'm sure there's a way to configure run on save though